packages/scalecube-microservice/src/api/Microservice.ts

```diff
@@ -1,9 +1,21 @@
 import { ProxyOptions, ServiceCall, CreateServiceCallOptions } from '.';
 
```

Comment:
```
/**
 * Microservice - provide functionality of microservice instance.
 *
 * `destroy` - The method is used to delete microservice and close all the subscriptions related with it
 *
 * `createProxy<T>: T` -  Create a proxy to a method and allowe extra logic when invoked.
 *
 * `createServiceCall` -  Expose serviceCall to use (not via Proxy)
 *
 */
```

Reply:
provides functionality of microservice container

Reply:
Creates a proxy to a method and provides extra logic when invoked

Reply:
Exposes serviceCall to a user (not via Proxy)